atlite/cutout.py

```diff
@@ -302,26 +310,31 @@ def transform_r(self):
 
     @property
     def dx(self):
+        """Spatial resolution on the x axis."""
         x = self.coords["x"]
         return round((x[-1] - x[0]).item() / (x.size - 1), 8)
 
     @property
     def dy(self):
+        """Spatial resolution on the y axis."""
```

Comment:
```suggestion
        """Spatial resolution on the y coordinate (latitude)."""
```

`y` is always implicitly latitude, right?

Reply:
see above

Reply:
Gotcha. Is 'coordinate' or 'axis' more comprehensive?